ExportToolYAMLs/wash.yaml

```diff
@@ -1,49 +1,7 @@
-planet_osm_line:
+wash_facilities:
   types:
     - lines
-  select:
-    - amenity
-    - blockage
-    - building
-    - covered
-    - depth
-    - diameter
-    - landuse
-    - layer
-    - man_made
-    - name
-    - natural
-    - pump
-    - tunnel
-    - water
-    - waterway
-    - width
-  where:
-    - 'waterway IS NOT NULL OR water=''reservoir'' OR natural=''water'' OR landuse=''reservoir'' OR man_made IN (''water_tower'',''pumping_station'') OR building=''pumping_station'' OR amenity = ''water_point'''
-planet_osm_polygon:
-  types:
     - polygons
-  select:
-    - amenity
-    - blockage
-    - building
-    - covered
-    - depth
-    - diameter
-    - landuse
-    - layer
-    - man_made
-    - name
-    - natural
-    - pump
-    - tunnel
-    - water
-    - waterway
-    - width
-  where:
-    - 'waterway IS NOT NULL OR water=''reservoir'' OR natural=''water'' OR landuse=''reservoir'' OR man_made IN (''water_tower'',''pumping_station'') OR building=''pumping_station'' OR amenity = ''water_point'''
```

Comment:
Pumping stations? Really? And no boreholes etc?

Reply:
its  - 'amenity IN (''water_point'',''drinking water'',''drinking_water'') OR emergency IN (''water_tank'',''water tank'')  OR natural=''spring'' '

Ithose were the place holders from the export tool. I some how didn't push my changes